markdown/developers/sync.md

```diff
@@ -3,26 +3,188 @@ title: Synchronisation
 subtitle: Learn how nf-core pipelines are automatically kept up to date with community standards.
 ---
 
-## Introduction
+# Introduction
 To keep all the nf-core pipelines up-to-date with the latest version of the community standards, we have implemented a synchronisation tool.
 This ensures that updates to the community standards are propagated to all the nf-core pipelines.
 
-If a pipeline is created using the `nf-core create` command, everything is automagically set up for the synchronisation to work.
-However, if a pipeline was not created using this command, this documentation will tell you how to set it up properly.
+There are three topics covered in this documentation page:
 
-## How it works
-In brief, when the synchronisation tool is triggered it compares every pipeline against the new standards and opens pull requests when mismatches are found.
-These pull requests needs to be manually dealt with by the pipeline maintainers.
+1. [Merging automated PRs](#merging-automated-prs)
+2. [Manual synchronisation](#manual-synchronisation)
+3. [Setting up a pipeline for syncing retrospectively](#setting-up-a-pipeline-for-syncing-retrospectively)
 
-Behind the scenes, the synchronisation tool fetches the variables needed for a pipeline and uses this to trigger a `nf-core create --no-git` command.
-The result from this command is then compared against what is stored in a special branch of the pipeline called the `TEMPLATE` branch.
-If any changes are detected, a pull request is created.
+### How template synchronisation works
+
+The `nf-core` helper tools have a subcommand for synchronising a pipeline with the nf-core template.
+Although this can be run manually, it is usually only used by the GitHub Actions automation:
+when a new version of nf-core/tools is released it runs for all nf-core pipelines and automatically opens pull-requests (PRs) with the necessary changes to every pipeline.
+These pull requests then need to be manually dealt with by the pipeline maintainers.
+
+Behind the scenes, this synchronisation is done by using `git`.
+Each repository has a special `TEMPLATE` branch which contains only the "vanilla" code made by the create tool.
+The synchronisation tool fetches the variables needed for a pipeline and uses this to trigger a `nf-core create --no-git` command with the latest version of the template.
+The result from this is then compared against what is stored in the `TEMPLATE` branch and committed.
+When merging from the `TEMPLATE` branch back in to the main `dev` branch of the pipeline, `git` should be clever enough to know what has been changed since the template was first used, and only present the relevant changes.
```

Comment:
```suggestion
When merging from the `TEMPLATE` branch back into the main `dev` branch of the pipeline, `git` should be clever enough to know what has changed since the template was first used, and therefore, it will only present the relevant changes.
```